tests/functional/clients/standalone/test_hints.py

```diff
@@ -18,52 +18,87 @@ def test_npm_hints(self, hints_image):
                     "version": "1.0rc",
                     "type": "npm",
                 },
+                {
+                    "name": "lodash",
+                    "location": "/node_modules/lodash/package.json",
+                    "version": "1.9.4",
+                    "license": "Not a real license",
+                    "type": "npm",
+                },
             ]
         }
         result = hints_image(hints, "npm")
         pkgs = result["image"]["imagedata"]["analysis_report"]["package_list"][
             "pkgs.npms"
         ]["base"]
+
+        # Verify message logged that package cannot be overwritten
+        assert "package already present under" in caplog.text
```

Comment:
Would be nice if you could add a flag to the parametrization that tells the test whether or not a given package should generate a log message. It seems that these tests might be annoying to update for someone who isn't familiar with the hints.